src/zenml/new/pipelines/build_utils.py

```diff
@@ -141,6 +141,20 @@ def reuse_or_create_pipeline_build(
                     Client().active_stack.name,
                 )
                 return existing_build
+            else:
+                logger.info(
+                    "Unable to find a build to reuse. When using a code, "
+                    "repository, a previous build can be reused when the "
+                    "following conditions are met:"
```

Comment:
```suggestion
                    "following conditions are met:\n"
```

Does this need a newline for formatting? (and subsequent lines as well?) I forget whether `logger.info` will format things by line. I have a feeling not, right? @coderabbitai wdyt?

Reply:
Ok so @schustmi if you're going to use bullet lists, maybe you want to add some newline breaks etc..

Reply:
Done @strickvl